yarn.lock

```diff
@@ -1324,7 +1324,7 @@ bucketclient@scality/bucketclient#8.1.0:
   dependencies:
     agentkeepalive "^4.1.3"
     arsenal scality/Arsenal#8ed8478
-    werelogs scality/werelogs#8.1.0
+    werelogs scality/werelogs#351a2a3
```

Comment:
I'm concerned here - we've pinned a commit that is *NOT* a tag. I'm not sure how that happened, it looks like we made a weird release.

```
git show 351a2a3

commit 351a2a3
Merge: 1a6e052 4e0d97c
Author: Katherine Laue <Katherine.laue@wustl.edu>
Date:   Fri Jun 28 14:03:48 2019 -0700

    Merge remote-tracking branch 'origin/w/7.5/improvement/S3C-2292-upgrade-nodejs' into w/8.0/improvement/S3C-2292-upgrade-nodejs
```